public/main.js

```diff
@@ -53,12 +53,21 @@ const autocompleteChannels =
 
 addEventListener('keydown', ({ key }) => {
  if (key === 'Escape') {
-  if (channelInputFocused) {
-   cancelChannelInput()
-   channelInput.blur()
-  } else {
-   channelInput.focus()
+  const expanded =
```

Comment:
Instead of doing this, can we set a global variable called `expandedElement` when an element is expanded, so we don't have to search for it.